packages/playwright/bundles/babel/roll.js

```diff
@@ -0,0 +1,23 @@
+const { chromium } = require('playwright');
+
+(async () => {
+  const browser = await chromium.launch();
+  const page = await browser.newPage();
+  const packageJSON = require('./package.json');
+  for (const dep of Object.keys(packageJSON.dependencies)) {
+    await page.waitForTimeout(3000);
+    console.log('Processing ', dep);
+    await page.goto(`https://www.npmjs.com/package/${dep}`);
```

Comment:
lets use `https://registry.npmjs.org/<pkg>/` instead?

Reply:
That's like 42Mb for Playwright :/